src/test/java/teammates/test/pageobjects/StudentFeedbackResultsPage.java

```diff
@@ -27,4 +27,20 @@ public String getQuestionAdditionalInfoButtonText(int qnNumber, String additiona
                 By.id("questionAdditionalInfoButton-" + qnNumber + "-" + additionalInfoId));
         return qnAdditionalInfoButton.getText();
     }
+
+    public boolean isExcludingSelfOptionVisible() {
+        return isElementVisible("excluding-self-check");
+    }
+
+    public void clickExcludingSelfCheckbox() {
+        click(By.id("excluding-self-check"));
```

Comment:
You may have multiple tables in the same page (with same id (which is already violating HTML5 standard)). Unless u ensure that there is only one such table in the HTML, u can do this. Nevertheless, it is not a good practice and may cause bugs in testing.

Reply:
Agreed. 

In fact, this now creates my new test to fail(commented out temporarily).
How would you suggest me to go about this because it's very much possible to have tables with this id on the same page and this problem existed before this PR as well.

Currently, all the tests check the entire HTML page is same or not, so should I add tests only for seeing if the checkbox is visible or not?

Reply:
@xpdavid waiting for input from your side on this?

Reply:
You can try CSS selector. I believe the checkbox is inside a element which u could identify by id.